src/plugins/Awesome.ts

```diff
@@ -8,15 +8,18 @@ interface AwesomeConfig extends JWPluginConfig {
 }
 
 export class Awesome extends JWPlugin {
-    constructor(dispatcher: Dispatcher, vDocument: VDocument, options?: AwesomeConfig) {
-        super(dispatcher, vDocument, options);
+    constructor(dispatcher: Dispatcher, options?: AwesomeConfig) {
+        super(dispatcher, options);
     }
 
     init(): Action {
         super.init();
         return {
-            type: ActionType.INSERT,
-            value: 42,
+            type: 'primitive',
+            name: 'insert',
+            payload: {
+                value: 42,
+            },
```

Comment:
Maybe we should remove this function (and/or this entire plugin) ? It makes no sense anyway. That being said, this PR would probably not be the right place to do so.

Reply:
I'd remove the plugin.